app-backend/batch/src/main/scala/stacExport/v2/CampaignExport.scala

```diff
@@ -630,8 +651,10 @@ class CampaignStacExport(
         case _ =>
           IO.pure(None)
       }
-      maybeCOGAssetAndLink: Option[(Tuple2[String, StacAsset], StacLink)] <- maybeNameAndIngestLocation match {
-        case Some((name, ingestLocation))
+      maybeCOGAssetAndS3Link: Option[(Tuple2[String, StacAsset], Tuple2[
```

Comment:
I didn't notice this when it was introduced and I'm happy to remove in my PR/rebase after yours, but we don't need the `Tuple2` type constructor, we can use the literal `(String, StacAsset)` instead